docs/guides/etherscan-voting.md

```diff
@@ -1,62 +1,81 @@
 
-3. If you're looking to vote on the last vote, take `votesLength - 1` as an id. If the `votesLength` is `89`, last vote would have the id `88`
-4. You can check the vote data with `getVote` method (number 6 on [Etherscan page])
+![](/img/etherscan-voting/getVote_2.png)
 
-![](/img/etherscan-voting/get-vote.png)
+### Step 3: Cast Your Vote
 
-[0x2e59a20f205bb85a89c53f1936454680651e618e]: https://etherscan.io/address/0x2e59A20f205bB85a89C53f1936454680651E618e
-[voting ui]: https://vote.lido.fi
-[etherscan]: https://etherscan.io/address/0x2e59A20f205bB85a89C53f1936454680651E618e#readProxyContract
-[contract/read as proxy]: https://etherscan.io/address/0x2e59A20f205bB85a89C53f1936454680651E618e#readProxyContract
-[etherscan page]: https://etherscan.io/address/0x2e59A20f205bB85a89C53f1936454680651E618e#readProxyContract
+- Navigate to the **Write as Proxy** tab.
+- Click **Connect to Web3** and connect the address where you hold LDO tokens. The indicator should turn green.
 
-## Voting
+![](/img/etherscan-voting/web3_connect_3.png)
+
+- Use the `vote` method (number 13).
+
+![](/img/etherscan-voting/vote_4.png)
+
+- Fill in the parameters `_voteId`, `_supports`, and `_executesIfDecided` and send the transaction:
+    - `_voteId` is the vote ID from Step 1.
+    - `_supports` indicates whether you support (`true`) or oppose (`false`) the vote.
+    - `_executesIfDecided` should be set to `false`.
+
+### Step 4: Sign the Transaction
+
+Sign the transaction to cast your vote. That's it! 🎉
+
+## Overriding
+
+### Step 1: Check Delegate's Vote
+
+- Go to the **Read as Proxy** tab of the Aragon Voting contract.
+- Use the `getVoterState` method (number 7).
+
+![](/img/etherscan-voting/getVoterState_5.png)
 
-1. Open "[Contract / Write as Proxy](https://etherscan.io/address/0x2e59A20f205bB85a89C53f1936454680651E618e#writeProxyContract)" tab on Etherscan
-2. Connect Etherscan UI to Web3 with either MetaMask or WalletConnect
+Enter the vote ID and your address to see how your delegate voted. 
 
-![](/img/etherscan-voting/connect-wallet.png)
+### Step 2: Vote Yourself
 
-3. Use method `vote` (number 6 on the [Etherscan Page](https://etherscan.io/address/0x2e59A20f205bB85a89C53f1936454680651E618e#writeProxyContract))
+If you disagree with the delegate's choice and wish to vote yourself, follow the steps in the **Voting Steps** section.
 
-![](/img/etherscan-voting/vote-1.png)
+## Delegating Through Etherscan
 
-- `_voteId` is the vote id from the point 2.
-- `_supports` is the flag of whether you're voting for (type `true`) or against (type `false`) the vote
-- `_executesIfDecided` is the flag to enact the vote if it could be executed right away in the tx sending the vote, `true` or `false`; from the experience of the previous votes, you may leave that as `false`
+### Assign a Delegate
 
-4. Fill in the parameters `_voteId`, `_supports` & `_executesIsDecided` and send the transaction
+1. Open the Aragon Voting contract.
```

Comment:
Assuming the initial steps have been completed, this step is redundant. It is also not present in all other flows, so I don't see a reason why it is present here. Maybe remove it?